pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -120,10 +135,11 @@ func (s *NodeController) SyncCloud() error {
 		nodeMap[node.Name] = &node
 	}
 
-	// Create or delete nodes from registry.
+	// Create nodes which have been created in cloud, but not in kubernetes cluster.
 	for _, node := range matches.Items {
 		if _, ok := nodeMap[node.Name]; !ok {
 			glog.Infof("Create node in registry: %s", node.Name)
+			node.Status.Phase = api.NodePending
```

Comment:
In the newer version, I removed node phase.

I'd like to have another PR handling phase update.  Currently all k8s components do not respect node phase, so I think we are good.